### PR TITLE
Don't throw on policy collision when policy names = *

### PR DESCRIPTION
**Pull request by Siegrift**

### CODE REVIEW

File: tests/trustedtypes_test.js [hunk 1]
```diff
@@ -562,6 +552,14 @@ describe('v2 TrustedTypes', () => {
       expect(TrustedTypes.getPolicyNames()).not.toContain('default');
       expect(TrustedTypes.getPolicyNames()).toContain('a');
     });
+
+    it('allows policy collision only when allowed policies includes *', () => {
```
Comment (koto) [inline tests/trustedtypes_test.js]: Can you create 2 separate test cases? One with `setAllowedPolicyNames(['*'])` and another one with `setAllowedPolicyNames(['foo'])` ?

Comment (koto) [inline tests/trustedtypes_test.js]: "policy collision" => "policy name collision"

Comment (Siegrift) [reply tests/trustedtypes_test.js]: done.

Comment (Siegrift) [reply tests/trustedtypes_test.js]: Yeah, good idea. I split the tests and also moved them in a separate `describe`.

**Pull request merged**